core/frontend/src/SpatialViewState.ts

```diff
@@ -256,6 +256,24 @@ export class SpatialViewState extends ViewState3d {
     this._treeRefs.setDeactivated(modelIds, deactivated, which);
   }
 
+  /** For getting the [TileTreeReference]s that are in the modelIds, for planar classification.
+   * @param modelIds modelIds for which to get the TileTreeReferences
+   * @param maskTreeRefs where to store the TileTreeReferences
+   * @internal
+   */
+  public setMaskRefs(modelIds: OrderedId64Iterable, maskTreeRefs: TileTreeReference[]): void {
```

Comment:
Is there are reason why this isn't a `getMaskRefs` returning an array instead of `setMaskRefs` that takes an array to populate? It's confusing, it's not actually "setting" anything.

Reply:
Could change the name to `collectMaskRefs`?
Did this because the array is created separately, and its existence is a flag, and this func may not get called.

Reply:
Sure. It's a nit, `setMaskRefs` just sounds like it's changing something about the ViewState.

Reply:
Renamed